### DISCUSSION TITLE
Unable to use {% process=false %} in next.js markdoc-starter

### DISCUSSION THREAD

**Discussion by andrewcmyk** [markdoc/markdoc#261] [Bug report] [answered]
I'm trying to document Liquid tags and having trouble ignoring/escaping variables. Example:

\```liquid {% process=false %}
{% i_want_this_to_be_plain_text %}
\```

Dev server output:
```
Warning: forwardRef requires a render function but was given undefined.
Warning: forwardRef requires a render function but was given undefined.
Invalid attribute: 'process'
```

**Comment by mfix-stripe** [comment, marked as answer]
Hey @andrewcmyk, sorry about this. You just need to add the `process` attribute, as I have done here: https://github.com/markdoc/markdoc-starter/commit/65c685cd787bb87ea9b11cfbfff46bc4dd19be3b

```js
attributes: {
  ...
  process: { type: Boolean, render: false, default: true },
},
```

This is now fixed in the latest starter.

**Comment by andrewcmyk** [comment]
Thanks for the quick action! I may be missing something still... now the code block won't render any contents at all.

**Reply by mfix-stripe** [reply]
Did you keep the other attributes in your `fence` config? You need the `content`, `language`, and `process` attributes 🙂

**Reply by andrewcmyk** [reply]
It took much longer than it should've for me to realize I could copy the `markdown-example` tag and code fence from the markdoc/docs repo to achieve what I was trying to do.
